src/Microsoft.AspNet.Hosting.Abstractions/IHostingEnvironment.cs

```diff
@@ -5,14 +5,27 @@
 
 namespace Microsoft.AspNet.Hosting
 {
+    /// <summary>
+    /// Provides information about the web hosting environment an application is running in.
+    /// </summary>
     public interface IHostingEnvironment
     {
+        /// <summary>
+        /// Gets or sets the name of the environment. This property is automatically set by the host to the value
+        /// of the <c>ASPNET_ENV</c> environment variable.
```

Comment:
Don't encourage the use of old environment variable names. We should just doc the current/new names.